documentation/operations/archival-process.md

```diff
@@ -0,0 +1,23 @@
+# Archival Process
+
+To archive the instances you will need to connect to the Ocim server using ssh and use the Django shell to archive instances since there is currently no UI to do so. Please, don't wait for the archival process for more than 5 minutes once it's running; this takes some hours to run, so it's better to check back later.
+
+*  Connect to Ocim using: `ssh ubuntu@ocim.net.opencraft.hosting`.
+*  See a list of screens with `screen -list`.
+*  The one that ends with "console" is what we want: `screen -r 1240..ocim` (or similar Ocim screen name).
+*  There are multiple screen windows in that session including one that runs the Ocim app in the foreground. Be careful not to fiddle with it or kill it by accident.
+*  You can switch around screens with CTRL+a -> SHIFT+" (press control and 'a' together, then shift and the double-quote key).
```

Comment:
```suggestion
*  You can switch around windows in screen with CTRL+a followed by SHIFT+" (press control and 'a' together, then shift and the double-quote key).
```

Reply:
I think 'You can switch around screens' is more accurate than 'You can switch around windows in screen' since you are not actually changing of window but changing screen

Reply:
-> replaced by followed by

Reply:
fixed windows in screen